JavaScript/packages/recognizers-date-time/src/dateTime/chinese/holidayConfiguration.ts

```diff
@@ -216,7 +216,7 @@ export class ChineseHolidayParser extends BaseHolidayParser {
         // get year (if exist)
         let year = referenceDate.getFullYear();
         let yearNum = match.groups('year').value;
-        let yearChinese = match.groups('yearchs').value;
+        let yearChinese = match.groups('yearCJK').value;
```

Comment:
Same.

Reply:
done

Reply:
I don't see the fix here. But I'll merge as-is as this is not a blocker.
Please change to the constants in the next PR.

Reply:
sorry, I missed this one, yes I will update in the next PR